backend/de.metas.manufacturing.rest-api/src/main/java/de/metas/handlingunits/rest_api/move_hu/MoveHURequest.java

```diff
@@ -2,17 +2,21 @@
 
 import de.metas.global_qrcodes.GlobalQRCode;
 import de.metas.handlingunits.HuId;
+import de.metas.handlingunits.QtyTU;
 import de.metas.handlingunits.qrcodes.model.HUQRCode;
 import lombok.Builder;
 import lombok.NonNull;
 import lombok.Value;
 
+import javax.annotation.Nullable;
+
 @Value
 @Builder
 public class MoveHURequest
 {
 	@NonNull HuId huId;
 	@NonNull HUQRCode huQRCode;
+	@Nullable QtyTU numberOfTUs;
```

Comment:
why not nonnull, default to ONE?

Reply:
bc It only makes sense when the hu is an aggregated one